devel/cquery/Portfile

```diff
@@ -22,15 +21,16 @@ long_description \
     cquery implements almost the entire language server protocol and provides some \
     extra features.
 
-platforms       darwin
-os.version      16.0.0
-homepage        https://github.com/cquery-project/cquery
+platforms          darwin
 
-version         20180302
+github.homepage    https://github.com/cquery-project/cquery
```

Comment:
The github.homepage command should be redundant and set by the proceeding gitHub.setup command. So just remove it.